### ISSUE TITLE
odfesql cannot cope with dots in index names

### ISSUE THREAD

**Issue by juergen-walter** [opendistro-for-elasticsearch/sql#1122]
odfesql cannot cope with dots in index names. Example query
```
odfesql> SELECT * from logs-json-2021.06.24
........ LIMIT 4;                                                                                                                                
``` 
results in the following 
```
error{'reason': 'Invalid SQL query', 'details': "Failed to parse query due to offending symbol [.06] at: 'SELECT * from logs-json-2021.06' <--- HERE... More details: Expecting tokens in {<EOF>, ';'}", 'type': 'SyntaxAnalysisException'}
```

I verified that I get no error if the index has no dots. From my point of view, that is a bug. Maybe I miss something ...

**Comment by juergen-walter** [comment]
The index name used above is a long time standard for logstash: https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-index

**Comment by chloe-zh** [comment]
Hi @juergen-walter thanks for reporting this issue, to run query in the plugin with the index that has special characters like dots, spaces etc. , you can quote the index name with backticks, for example:
```
odfesql> SELECT * from `logs-json-2021.06.24`
........ LIMIT 4; 
```
And this should be working well. 

Btw, our project has been fully migrated to the new repository https://github.com/opensearch-project/sql, feel free to create issues in our new repo so we are able to see it at the first place. Thanks!

Ref: https://github.com/opendistro-for-elasticsearch/sql/blob/develop/docs/user/general/identifiers.rst#delimited-identifiers

**Comment by juergen-walter** [comment]
@chloe-zh thank you very much for your reply. I just learned that this is an SQL standard. Cool, that version 1.0 of OpenSearch is out, I already followed the news!

I looked into odfesql based on the following documentation:
https://opendistro.github.io/for-elasticsearch-docs/docs/sql/cli/
especially this gif
https://opendistro.github.io/for-elasticsearch-docs/docs/images/cli.gif
--> you might want to link this information there or make it more prominent
